### DISCUSSION TITLE
Welcome to DogeRat Discussions!

### DISCUSSION THREAD

**Discussion by shivaya-dav** [shivaya-dav/DogeRat#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Thrisia82** [comment]
Hello my name is Trish. I need help

**Reply by Chaitu1710** [reply]
what help do you need ??
